frontend/src/components/VPopover/VPopover.vue

```diff
@@ -15,6 +15,7 @@ import type { Placement, Strategy } from "@floating-ui/dom"
 
 const props = withDefaults(
   defineProps<{
+    id: string
```

Comment:
```suggestion
    /**
     * The id used to keep track of the modal in the open modal stack, to enable
     * nested modals.
     */
    id: string
```

Should this have a similar comment?